conda_smithy/configure_feedstock.py

```diff
@@ -791,12 +791,13 @@ def dump_subspace_config_files(
             package_key(config, top_level_loop_vars, metas[0].config.subdir),
         )
         short_config_name = config_name
+        salt = hashlib.sha256(config_name.encode("utf-8")).hexdigest()[:8]
+        # drone has a limit of 50, see https://github.com/conda-forge/conda-smithy/issues/1188
         if len(short_config_name) >= 49:
-            h = hashlib.sha256(config_name.encode("utf-8")).hexdigest()[:10]
-            short_config_name = config_name[:35] + "_h" + h
-        if len("conda-forge-build-done-" + config_name) >= 250:
+            short_config_name = config_name[:40] + "_h" + salt
+        if len("conda-forge-build-done-" + config_name) >= 200:
```

Comment:
```suggestion
        if len(config_name) >= 190:
```

Reply:
Doesn't this need to take into account something like
```
C:\Users\runnerx\_work\pytorch-cpu-feedstock\pytorch-cpu-feedstock\.ci_support\
```
(not sure where you [got that](https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6967#issuecomment-2614846418) from, and why the `pytorch-cpu-feedstock` is doubled), then we can directly check `>=260`.

Reply:
This is done now